thyra-installer.bat

```diff
@@ -0,0 +1,60 @@
+@echo off
+
+curl -L https://github.com/massalabs/thyra/releases/latest/download/thyra-server_windows_amd64 --output thyra-server.exe
+
+SET ACRYLIC_PATH="C:\Program Files (x86)\Acrylic DNS Proxy"
+@REM %~dp0 is alias for current working directory
+SET STARTING_WORKING_DIR=%~dp0
+
+IF NOT EXIST %STARTING_WORKING_DIR%thyra-server.exe (
+    ECHO "The installer couldn't download Thyra from Github. Please retry or check your internet connection."
+    PAUSE
+    EXIT
+)
+
+IF NOT EXIST %ACRYLIC_PATH%\AcrylicHosts.txt (
+    curl -L https://sourceforge.net/projects/acrylic/files/Acrylic/2.1.1/Acrylic-Portable.zip/download --output Acrylic-Portable.zip
+    IF NOT EXIST %STARTING_WORKING_DIR%Acrylic-Portable.zip (
+        ECHO "The installer couldn't download DNS Acrylic Proxy. Please retry or check your internet connection."
+        PAUSE
+        EXIT
+    )
+
+    MKDIR %ACRYLIC_PATH%
+    IF NOT EXIST %ACRYLIC_PATH%\ (
+        ECHO "Couldn't create DNS Acrylic Proxy installation folder. Please execute this script as an administator."
+        PAUSE
+        EXIT
+    )
+    MOVE %STARTING_WORKING_DIR%Acrylic-Portable.zip %ACRYLIC_PATH%\
+    tar -xf  %ACRYLIC_PATH%\Acrylic-Portable.zip -C %ACRYLIC_PATH%
+    @REM Ideally we should check here that Acrylic has been successfully unarchived.
+    CD %ACRYLIC_PATH%
+
+    CALL "InstallAcrylicService.bat"
+
+    @REM Allows for variable value assertion inside for loops at runtime
+    @REM This for loop iterate over all connected network adapters and set their DNS.
```

Comment:
Does that mean that I'll have the DNS setup for my Ethernet & Wifi ( for instance ) ?

Reply:
If both Wi-Fi and Ethernet network adapters are connected at the moment you execute the script, then yes it will setup the DNS for both. If only one of them is connected DNS will be setup for this one only.

Reply:
Why not configuring all network adapters ?

Reply:
If I have several network adapters and I don't want all of them to be using Acrylic DNS I should have the possibility to avoid this.
I don't really see why we should force this on the user because IMHO it's not our choice to make.
Although this choice could take the shape of a flag given to the script (Only connected net adapters / All adapters).

Reply:
Don't you think that it can be disturbing to have thyra working on ethernet, but not in wifi ?

Reply:
It will be for sure if people are not aware of how it works but it could be stated as a disclaimer in the installation documentation process.
On another hand we could configure all of them and the disclaimer could state that ALL network adapters will be configured to work with Acrylic DNS, connected or not.
The thing I am afraid of here is that it works great for madame Michu but may break things for more advanced users who have different DNS for different adapters (even though I don't really see the use case for this).